docs/container-contract.md

```diff
@@ -39,16 +39,16 @@ controller performs an anonymous lookup of the image.
 
 For example, consider the following `Task`, which uses two images named
 `gcr.io/cloud-builders/gcloud` and `gcr.io/cloud-builders/docker`. In this example, the
-Pipelines controller retrieves the `entrypoint` value from the registry, which allows
-the `Task` to execute the `gcloud` and `docker` commands, respectively.
+Pipelines controller retrieves the `entrypoint` value from the registry, which means
+the container will execute using the default `entrypoint` defined in the image.
 
 ```yaml
 spec:
   steps:
     - image: gcr.io/cloud-builders/gcloud
-      command: [gcloud]
+      # No command specified, so the default entrypoint from the image will be used
     - image: gcr.io/cloud-builders/docker
-      command: [docker]
+      # No command specified, so the default entrypoint from the image will be used
```

Comment:
Maybe also mention that gcloud and docker ARE the default entrypoints for the respective images.